numpy/random/src/distributions/distributions.c

```diff
@@ -1,9 +1,27 @@
-#include "numpy/random/distributions.h"
+
+/*
+ * RAND_INT_TYPE is used to share functions between _generator.pyx and
+ * legacy mtrand.pyx (where it is expected to be "long").
+ *
+ * If you change a function in this file, then the original unmodified
+ * copy must be retained for use in RandomState via mtrand.pyx by copying to
+ * the legacy-distributions.c source file.
+ */
+
+#ifndef RAND_INT_TYPE
+  #define RAND_INT_TYPE int64_t
+  #define RAND_INT_MAX INT64_MAX
+  #include "numpy/random/distributions.h"
+#else
+  /* For internal numpy legacy use only */
+  #include "include/distributions-long.h"
+#endif
+
 #include "ziggurat_constants.h"
 #include "logfactorial.h"
 
 #if defined(_MSC_VER) && defined(_WIN64)
-#include <intrin.h>
+  #include <intrin.h>
 #endif
 
 /* Inline generators for internal use */
```

Comment:
This file is formatted with 2-space tabs instead of 4-space. Is now a good time to remedy that?

Reply:
I am happy if you do that (just ping Kevin on it before merging?), should be its own pure STY PR though (and if possible cover other files in random if they exist?). Not sure how others feel about such fixups, but I tend to feel that not doing style cleanups is also not a solution...